### DISCUSSION TITLE
New Unguided Level - Trash Church

### DISCUSSION THREAD

**Discussion by Moburma** [swfans/syndwarsfx#181] [Show and tell]
Well, 98% new anyway. So I took the barest bones of the original barely started Bullfrog level and built something new out of it. I think it's at a point where it's maybe 60%+ done, so I thought I'd get some much needed feedback on it. This is meant to be a tough near-endgame level so expect to be.... challenged.

Download here:

https://filebin.net/7j8okc0jm4adba34

To use it, copy over the existing level file c036l004.dat and run with  -m 2,69

I need to actually tweak the objectives, but they are broadly correct out of the box. The sole big change is you are required to rendezvous with a crack team of your fellow unguided who are hiding out in the bar/restaurant near the start shown below. Just head near the door and they will join you. 

![](https://i.ibb.co/y42fpxH/Desktop-Screenshot-2024-06-22-17-10-59-45.png)

The overall objective is to destroy the church and kill all zealots. I will provide proper objectives later.

Hints/notes:

* You start off with weak weaponry, keep a close eye on what enemies are carrying and upgrade ASAP from them
* Head to the small temple first
* Don't summon the other unguided until you've got the car situation under control...
* Elite zealots = high risk, high reward
* I intend to add some items into the level. At the minute that will likely be 2x automedikits in the Police station, and something in the small temple like a nuclear grenade. Not there yet
* The climactic battle is not done yet, I will make it more interesting
* I roughly worked out the level into 9 Squares split by roads, with each one having something interesting in. The middle and top squares aren't done yet, I will put something in them
* The top area will likely be some kind of landing field for zealot vehicles, I might do a little starting cutscene of elite zealots landing in the big shuttle and walking into the church
* Generic civilians are not added yet, I will put some in later

Feedback wanted:
Good and bad bits, difficulty, bugs, etc

It's meant to be pretty hard and err, it is, especially early on. I just did a playthrough a minute ago, and beat it with only one single character left, was actually quite exciting! But let me know if it's too much so. Perhaps I would remove one of the roaming zealot cars, they are quite brutal and can be frustrating to deal with.

**Comment by geist22** [comment]
Got it on second attempt. Vehicles with Launchers are kinda cruel when your guys are so slow and squishy. Accidentally triggered the friendy Unguided early and got them killed by the vehicles, but since one of them left a 3rd LR Rifle behind, I wasn't too sad about losing them. Didn't expect the flamethrower on the civilians in the middle.

Some notes:
1. The Zealots following the High Priest in the district next to your spawnpoint have lasers. Oversight? They probably should have Electron Maces instead, as the Church never uses lasers in the base campaigns.
2. Same guys, when the High Priest is killed they just stop and stand around, waiting to get LR'd down.
3. The Zealots in front of the final church don't act as a team, you can pull them individually. Feels too easy.

From my point of view, the whole level is about the vehicles with the Launchers, and once they're destroyed it's basically over. Since your Unguided are so slow, using them as a full team will always result in one guy getting a direct hit from any fired missile, and they die in two. So I just used the LR rather than engaging the vehicles, as they were too dangerous. Going with a single person with a Plasma Lance worked for a bit, but the moment a second vehicles gets near that strategy falls out of the window.

To be honest I'm really not a fan of all the difficulty of the mission being concentrated in a single element like this. I'd rather have the vehicles less dangerous (having faster/tankier units would be enough, see Church mission 16) and the enemy groups stronger for a challenge that's more evenly distributed throughout the level.

**Reply by Moburma** [reply]
> Some notes:
> 
>     1. The Zealots following the High Priest in the district next to your spawnpoint have lasers. Oversight? They probably should have Electron Maces instead, as the Church never uses lasers in the base campaigns.

I actually can't remember doing that, will fix.

>     2. Same guys, when the High Priest is killed they just stop and stand around, waiting to get LR'd down.

Yeah, I didn't bother putting in contingency behaviour, I would have thought they'd be close enough to attack you if he died, but I will put in an until clause on his death to attack

>     3. The Zealots in front of the final church don't act as a team, you can pull them individually. Feels too easy.

Yes, this just isn't started yet. I just made a cool looking basic setup and left it. It needs lots more work, it's far too easy and a bit boring, also very static. The multiplayer mission c036l005.dat is actually the only level that uses this church as a zealot stronghold and that has loads of movement and seems very dynamic, so I will use that as inspiration. Also, as I discovered, claw mechs are bugged so can't be placed to face a certain way in a level (they are meant to be facing away from the church), but nevermind. I'd probably have the finale at least involving elite zealots with plasma lances.

> To be honest I'm really not a fan of all the difficulty of the mission being concentrated in a single element like this. I'd rather have the vehicles less dangerous (having faster/tankier units would be enough, see Church mission 16) and the enemy groups stronger for a challenge that's more evenly distributed throughout the level.

Yeah, this aligns with my thinking. About 80% of my deaths playing it come from them and it gets boring as aside from being frustrating, the best tactic to deal with them is just waiting around, which is never going to be very exciting. They were one of the first things I put in, as they are the only Bullfrog placed enemies in the level (well there is only one, so I expanded on it). 

I think they should either be:
a) Just removed entirely
b) start off invisible and maybe appear later (once the player has a plasma lance?)

I need to put something in the empty areas of the level. I think I need to beef up existing encounters as well, and try and put some tricky/ambush type situations in as it's a bit too rote at the moment.

It's suprisingly hard to balance this game, as there is so much interplay of stats. The easiest thing I've found is using body mods, these seem to work as the best benchmark for how tough enemies are. Most in the level are full level 2 set.

Also the flamethrower sequence shows how bugged and crap the flamer is - there's about a 2/5 chance that the zealot will try to burn them with it but will somehow miss for absolutely ages at point blank range. I'm pretty sure there's just something fundamentally wrong with it, which is why it's so useless.

**Reply by mefistotelis** [reply]
> I think they should either be:
a) Just removed entirely
b) start off invisible and maybe appear later (once the player has a plasma lance?)

What about one of the Unguided planting an Ion Mine before joining you?
One of the vehicles will get auto-eliminated, and you have to just make sure your team is far enough.
The vehicle can be asked to drive to the mine to make sure it gets blown in short time. For the situation where the planter gets shot before planting - it should be up to the player to get close to the Unguided at correct moment.

**Reply by geist22** [reply]
Another thing I forgot to mention before: giving the allied Unguided desirable weapons (the LR Rifle in particular, but the Launcher can also be considered to a lesser extent) can make deliberately getting them killed an attractive tactic. Having them use Miniguns/Lasers but being more durable (or getting more reinforcements in different locations) might be something to consider.

For the Launcher specifically, the Unguided seem to have very low energy, judging from my short experience. After shooting a few times they seem to be out of energy and just don't do anything anymore for a while.

**Reply by Moburma** [reply]
> What about one of the Unguided planting an Ion Mine before joining you?

Not a bad idea, but I think it would be too fiddly (plus from experience, making NPCs do stuff when they potentially under fire is actually quite hard, see the weird stuff that happens in the 'Slott' cut scene as an example), plus you've got no control over what cars are coming when.

This actually inspired me to do something else though - start one of the player punks with ion mines. This is actually something I've wanted to do as a) I think asymetric weapon load outs are interesting to play with b) I personally feel Ion mines are one of many weapons that are quite neglected. I hardly ever use them as you get them so late, and while they help take health off tanks as a stand off weapon before you have plasma lances, they do so little damage each it's barely even worth the effort. Trying this now I think is quite fun and stops this part being so obnoxious (if still quite hard). Even if you kill one car with them (they do about 80 - 90% damage on hit) you've then got a launcher to make the next one easier. 

Still not sure if the cars should just go entirely, but this is making it enjoyable for now.

>Another thing I forgot to mention before: giving the allied Unguided desirable weapons (the LR Rifle in particular, but the Launcher can also be considered to a lesser extent) can make deliberately getting them killed an attractive tactic. Having them use Miniguns/Lasers but being more durable (or getting more reinforcements in different locations) might be something to consider

This is actually somewhat on purpose, I wanted them to have more of a disposable feel, as they're probably going to get wasted anyway, so there is that choice to kill them yourself. 
As an aside I think this is one thing that is hard to grasp the full intention behind the unguided, it seems to me in many ways they're not really the "good guys" other than by default because the other factions are so monstrous. They seem very much to work on the ends justifying the means, as they do things like massacre civilians all through the game. 

>For the Launcher specifically, the Unguided seem to have very low energy, judging from my short experience. After shooting a few times they seem to be out of energy and just don't do anything anymore for a while.

That is a good point, maybe I will swap for another LR rifle or something. I didn't want them to just be lasers or weaker though, I wanted them to be more of a game changer. I guess I could also just boost their energy levels.

**Comment by Moburma** [comment]
I've kept working on this one, so here's a new release:
https://filebin.net/14ndcp5dyg093tqn

Some new enemies and surprises. Ion mines are in. Did a lot more work on the climax of it, that's the bit  I want to concentrate on.

As ever, tell me what you think.

**Reply by geist22** [reply]
I'm really not a fan of the Zealot group and the High Priest in that one temple just spawning in. In general, enemies spawning in when they don't have to feels too artifical, and the High Priest is also missable as you have to get pretty close for him to spawn. I only moved there because one was there in the older version of the level, so I sort of already expected something. Ambush spawning in after the High Priest died is ok, since surprise is kind of the point of ambushes.

Giving every Unguided a full stack of Ion Mines feels overkill. I ended up using only two, so just one Unguided with them would feel more appropriate.

The mechs at the end can be sniped with the LR Rifle without them aggroing, not sure if that's intended.

Also having the issues with enemies moving back and forth here, too.

Other than that, looks pretty good.

**Reply by Moburma** [reply]
Thanks. I made a new version based on above feedback:
https://filebin.net/3f4mmp5e7dv3u348

You're right on the elite zealot thing - I originally intended to have an item in the temple so it made sense to go inside, but ended up putting them in the Police station instead, so it made zero sense why the player would go in there. That's the problem when you do something out of habit, you don't realise it's nonsensical! I changed that so the zealots inside are always visible. I also reduced it down to one unguided with ion mines (I actually originally had this, then change it to all at the last minute as they're not that useful, but that does make the cars totally trivial).

I tweaked the final mech activation area, but to be honest not meaningfully tested it. Also changed the starting camera angle to actually be useful. Probably finished for now.

Anyone lurking? Have a play and say what you think.
